plugins/home/api-report.md

```diff
@@ -84,20 +84,10 @@ export const HomepageCompositionRoot: (props: {
   children?: ReactNode;
 }) => JSX.Element;
 
-// Warning: (ae-missing-release-tag) "homePlugin" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
-export const homePlugin: BackstagePlugin<
-  {
-    root: RouteRef<undefined>;
-  },
-  {}
->;
-
-// Warning: (ae-missing-release-tag) "RandomJokeHomePageComponent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+// Warning: (ae-missing-release-tag) "HomePageRandomJoke" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
 //
 // @public (undocumented)
-export const RandomJokeHomePageComponent: ({
+export const HomePageRandomJoke: ({
```

Comment:
I have no idea what the usage of this component actually is in real Backstage instances, but I think changing the name technically constitute a minor.

Given how nascent the plugin is (and how unlikely it is anyone would actually be using this in a real instance of Backstage), you could probably get away with a heavy **NOTE:** in the changeset.

Reply:
Sounds good!

Fixed in 03de8d2.